### PR TITLE
License update

### PR DESCRIPTION
**Pull request by aggrolite**
Hi,

I was requesting a change to the reddit wiki [here](https://github.com/reddit/reddit/issues/1770) and noticed some issues with the geddit license body:

1. Date range should be maintained to extend the lifetime of a copyright (warning: I am not a copyright expert).
2. Copyright holder was absent from the license body.
3. The 3rd clause lacked reference to the copyright holder. I've used [this](https://opensource.org/licenses/BSD-3-Clause) page as a guideline.

Let me know if I stated anything inaccurate and I'll be happy to update this PR and/or the reddit wiki issue.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -8,14 +8,14 @@ modification, are permitted provided that the following conditions are met:
...
-    * Neither the name of the <organization> nor the
+    * Neither the name of the copyright holder nor the
       names of its contributors may be used to endorse or promote products
       derived from this software without specific prior written permission.
 
 THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
 ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
 WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
-DISCLAIMED. IN NO EVENT SHALL <COPYRIGHT HOLDER> BE LIABLE FOR ANY
+DISCLAIMED. IN NO EVENT SHALL <Jimmy Zelinskie> BE LIABLE FOR ANY
```
Comment (aggrolite) [inline LICENSE]: Alternatively, this can be rewritten as "IN NO EVENT SHALL THE COPYRIGHT HOLDER BE LIABLE..."

Comment (jzelinskie) [inline LICENSE]: i prefer having the text COPYRIGHT HOLDER such that if the copyright holders change (more are added) the contents of the license need not be changed.

Comment (aggrolite) [inline LICENSE]: The page I linked to actually says "IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE..." which I think is more specific.

**Comment by aggrolite** [comment]
See updated diff which rephrases liability. Thoughts?

**Comment by jzelinskie** [comment]
Looks great! Thanks again!

**Pull request merged**